mod/groups/views/default/groups/listing/discussion.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
I suppose we will be moving the tab registration over to the discussion plugin eventually, and we have this view in the groups plugin, we will be stuck with it. I will move to discussions.